libmscore/tie.cpp

```diff
@@ -45,6 +45,11 @@ void TieSegment::draw(QPainter* painter) const
       if (tie()->endNote() && tie()->endNote()->chord()->crossMeasure() == CrossMeasure::SECOND)
             return;
 
+      // hide tie in tablature if we are not showing back-tied fret marks
+      StaffType* st = staff()->staffType(tick());
+      if (st->isTabStaff() && !st->showBackTied())
```

Comment:
It is clear why the decision on hiding a fret marking can't be currently done on note layout stage, but does it make sense for ties? Wouldn't it be better just to set an empty `bbox` to its segments or not create them at all on layout in this case?

Reply:
Yes, you are probably right. My first thought was to hide the tie in the same way that the fret is hidden, and since it worked, I didn't give it a second thought.

Reply:
I modified this PR to not layout the ties in the first place, rather than hiding them in TieSegment::draw().

Reply:
OK, thanks!